src/commands/app/db/collection/insertMany.js

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright 2025 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { DBBaseCommand } from '../../../../DBBaseCommand.js'
+import { Args, Flags } from '@oclif/core'
+import chalk from 'chalk'
+import { isNonEmptyString } from '../../../../utils/inputValidation.js'
+
+export class InsertMany extends DBBaseCommand {
+  async run () {
+    const { collectionName, documents } = this.args
+    const { bypassDocumentValidation } = this.flags
+
+    try {
+      this.log(chalk.blue(`Inserting ${documents.length} documents into collection '${collectionName}'...`))
+
+      // Build options object
+      const insertOptions = {}
+      if (bypassDocumentValidation) {
+        insertOptions.bypassDocumentValidation = true
+        this.log(chalk.dim('   Bypassing document validation'))
+      }
+
+      const client = await this.db.connect()
+      const collection = await client.collection(collectionName)
+
+      // Perform the insertMany operation
+      const result = await collection.insertMany(documents, insertOptions)
+
+      this.debugLogger?.info?.('Documents inserted successfully:', result)
+
+      const response = {
+        collectionName,
+        insertedCount: result.insertedCount,
+        insertedIds: result.insertedIds,
```

Comment:
Inserted IDs are instances of ObjectId, for readability/usability I'd prefer they were replaced with `id?.toString()` rather than the raw object.